### PR TITLE
added snippets in bubbleSort

### LINKED ISSUE #46: Add snips in `src/BubbleSort_array.md` for other languages

### PR DESCRIPTION
**Pull request by MrSingh2000**
closes #46 
snippets for bubble sort are added for other languages including cpp, java, go, php, etc.
No comments were made and function names and parameters were kept same.

### CODE REVIEW

**Comment by MrSingh2000** [comment]
@mention-bot This pull request is related to #46

File: src/BubbleSort_array.md [hunk 1]
```diff
@@ -35,3 +44,150 @@ function bubbleSort(A) {
...
+        for j in range(0, n-i-1):
+            if arr[j] > arr[j+1]:
+                arr[j], arr[j+1] = arr[j+1], arr[j]
+```
+
+### java
+
+```java
+public class BubbleSort {
+    public static void bubbleSort(int[] arr) {
```
Comment (rahulrk-dev) [inline src/BubbleSort_array.md:86]: Just keep it a simple function, instead of class

**Review by rahulrk-dev — CHANGES_REQUESTED** [review]

File: src/BubbleSort_array.md (hunk 1 above)
Comment (MrSingh2000) [reply src/BubbleSort_array.md]: Done ♥

**Review by rahulrk-dev — APPROVED** [review]
Thanks @MrSingh2000, merging

**Pull request merged**